fact/analysis/statistics.py

```diff
@@ -1,6 +1,46 @@
 import numpy as np
 
 
+def power_law(energy, phi_0, gamma):
+    r'''
+    Simple power law
+
+    .. math::
+        \phi = \phi_0 \cdot E ^ \gamma
+
+    Parameters
+    ----------
+    energy: number or array-like
+        energy points to evaluate
+    phi_0: float
+        Flux normalization
+    gamma: float
+        Spectral index
+    '''
+    return phi_0 * energy**(-gamma)
```

Comment:
this -(minus) is maximum surprise, no? 

I don't think putting the - into the docu is good either, the user should give -2.7 it she means -2.7